.bazelrc

```diff
@@ -91,21 +91,50 @@ build:coverage --javabase=@bazel_tools//tools/jdk:remote_jdk11
 #############################################################################
 # Experimental EngFlow Remote Execution Configs
 #############################################################################
-# remote-ci-macos: Flags to run CI using Engflow's remote execution. This
-# config is used for MacOS-specific flags.
+# remote-ci-common: These options are valid for any platform, use the configs below
+# to add platform-specific options. Avoid using this config directly and
+# instead use a platform-specific config
 #############################################################################
 # --config=remote sets --auth_enabled=true to enable GCP authentication,
-# however, the MacOS remote execution cluster doesn't use GCP auth
-# so we have to override it to 'false' here.
-build:remote-ci-macos --config=remote
-build:remote-ci-macos --xcode_version_config=//ci:xcode_config
```

Comment:
looks like we lost this config in this change?

Reply:
Good catch! Done now